src/main/java/com/kilopo/kosshop/util/HibernateUtil.java

```diff
@@ -1,13 +1,43 @@
 package com.kilopo.kosshop.util;
 
+import com.kilopo.kosshop.entity.Address;
+import com.kilopo.kosshop.entity.BaseEntity;
+import com.kilopo.kosshop.entity.Category;
+import com.kilopo.kosshop.entity.Order;
+import com.kilopo.kosshop.entity.OrderProduct;
+import com.kilopo.kosshop.entity.Producer;
+import com.kilopo.kosshop.entity.Product;
+import com.kilopo.kosshop.entity.User;
 import org.hibernate.SessionFactory;
 import org.hibernate.boot.registry.StandardServiceRegistryBuilder;
 import org.hibernate.cfg.Configuration;
 
+import java.util.Properties;
+
 public class HibernateUtil {
 
     public static SessionFactory getSessionFactory() {
-        Configuration cfg = new Configuration().configure();
+        Properties properties = new Properties();
+        properties.setProperty("hibernate.connection.driver_class", "org.postgresql.Driver");
+        properties.setProperty("hibernate.connection.url", "jdbc:postgresql://localhost:5432/kosshop");
+        properties.setProperty("hibernate.connection.username", "postgres");
+        properties.setProperty("hibernate.connection.password", "root");
+        properties.setProperty("hibernate.show_sql", "true");
+        properties.setProperty("hibernate.dialect", "org.hibernate.dialect.PostgreSQL82Dialect");
+        properties.setProperty("hibernate.hbm2ddl.auto", "create");
+        properties.setProperty("hibernate.enable_lazy_load_no_trans", "true");
```

Comment:
change prop names to enum (ex. "hibernate.connection.url" to Environment.USER). Just import it.